connectsdk/build.gradle

```diff
@@ -103,11 +64,8 @@ dependencies {
     compile "com.android.support:support-v4:${rootProject.ext.appCompatVersion}"
     compile "com.android.support:appcompat-v7:${rootProject.ext.appCompatVersion}"
     compile "com.android.support:mediarouter-v7:${rootProject.ext.appCompatVersion}"
-    compile 'com.google.android.gms:play-services-cast:9.8.0'
+    compile 'com.google.android.gms:play-services-cast:10.0.1'
```

Comment:
Let's hope this works. ConnectSDK hasn't been updated in ages and there is a new Google Cast API.